lib/rubocop/cop/cop_store.rb

```diff
@@ -0,0 +1,159 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Error raised when an unqualified identifier is used that could
+    # refer to two or more cops under different namespaces
+    class AmbiguousCopName < RuboCop::Error
+      MSG = 'Ambiguous cop name `%<name>s` used in %<origin>s needs namespace' \
+            ' qualifier. Did you mean %<options>s?'.freeze
+
+      def initialize(identifier, origin, namespaces)
+        options = namespaces.map { |namespace| "#{namespace}/#{identifier}" }
+        message = MSG % {
```

Comment:
This also shows how we can improve our `%` detection cop - it should check for a hash literal as it's argument. :-)